installer/conf/container.conf

```diff
@@ -60,11 +60,35 @@
 	]
 </source>
 
+# Container host inventory
+<source>
+	type omi
+	run_interval 60s
+	tag oms.api.ContainerNodeInventory
+	items [
+		["root/cimv2","Container_HostInventory"]
+	]
+</source>
+
 # Filter for correct format to endpoint
 <filter oms.container.**>
 	type filter_container
 </filter>
 
+<match oms.api.ContainerNodeInventory**>
+  type out_oms_api
+  log_level debug
+  buffer_chunk_limit 20m
+  buffer_type file
+  buffer_path %STATE_DIR_WS%/out_oms_containerinventory*.buffer
+  buffer_queue_limit 20
+  flush_interval 20s
+  retry_limit 10
+  retry_wait 15s
+  max_retry_wait 9m
+</match>
+
+
```

Comment:
Be sure to test these numbers and be sure that the agent has no problem keeping up with a reasonable expected load.